core/examples/luigi-sample-angular/src/luigi-config/extended/navigation.js

```diff
@@ -151,58 +194,79 @@ class Navigation {
      */
     fallbackLabelResolver: id => id.replace(/\b\w/g, l => l.toUpperCase())
   };
-  // The following configuration will be used to render a product switcher component
-  productSwitcher = {
-    items: [
+
+  getProductSwitcherItems = () => {
+    const items = [
       {
         icon: 'https://sap.github.io/fundamental/images/products/06.png',
         label: 'hybris',
         externalLink: {
           url: 'https://www.hybris.com',
           sameWindow: false
         }
-      },
-      {
+      }
+    ];
+    if (projectExists('pr1')) {
+      items.push({
         icon: 'https://sap.github.io/fundamental/images/products/06.png',
         label: 'Project 1',
         link: '/projects/pr1'
-      },
-      {
+      });
+    }
+    if (projectExists('pr2')) {
+      items.push({
         icon: 'https://sap.github.io/fundamental/images/products/06.png',
         label: 'Project 2',
         link: '/projects/pr2'
-      },
-      {
+      });
+    }
+    if (projectExists('pr3')) {
+      items.push({
         icon: 'https://sap.github.io/fundamental/images/products/06.png',
         label: 'Project 3',
         link: '/projects/pr3'
-      }
-    ]
+      });
+    }
+    return items;
   };
 
-  profile = {
-    logout: {
-      label: 'End session'
-      // icon: "sys-cancel",
-    },
-    items: [
+  // The following configuration will be used to render a product switcher component
+  productSwitcher = {
+    items: this.getProductSwitcherItems
+  };
+
+  getProfileItems = () => {
+    const items = [
       {
         label: 'Luigi in Github',
         externalLink: {
           url: 'https://github.com/SAP/luigi',
           sameWindow: false
         }
-      },
-      {
+      }
+    ];
+    if (projectExists('pr1')) {
+      items.push({
         icon: '',
         label: 'Project 1',
         link: '/projects/pr1'
-      },
-      {
+      });
+    }
+    if (projectExists('pr3')) {
```

Comment:
A bit confusing since pr2 is missing here, please add it.

Reply:
Sure, I understand you find it a little bit confusing, I will add it.